py3status/modules/sysdata.py

```diff
@@ -274,9 +292,20 @@ def _get_cputemp(self, zone, unit):
 
         return cpu_temp
 
+
     def sysdata(self):
         sys = {"max_used_percent": 0, "temp_unit": self.temp_unit}
 
+        cores = None
+        if self.init["cpu_freq_max"]:
+            cores = self._get_cpuinfo()
+            sys["cpu_freq_max"] = max(cores) / 1000  # convert to GHz
+
+        if self.init["cpu_freq_avg"]:
+            if cores is None:
+                cores = self._get_cpuinfo()
+            sys["cpu_freq_avg"] = (sum(cores) / len(cores)) / 1000
```

Comment:
Also implemented in 33d1981. I went with `cpu` for the threshold color, but `load` could also make sense?

Reply:
I actually think you should have a new threshold, not reuse existing one, because it would not be possible to define a threshold for both CPU usage percentage _and_ frequencies as the values are in different range... I think the only option is to use new `cpu_freq` threshold.